sycl/plugins/hip/CMakeLists.txt

```diff
@@ -13,7 +13,20 @@ if(NOT EXISTS "${SYCL_BUILD_PI_HIP_ROCM_DIR}")
 endif()
 
 # Set HIP include dirs
-set(HIP_HEADERS "${SYCL_BUILD_PI_HIP_ROCM_DIR}/hip/include;${SYCL_BUILD_PI_HIP_ROCM_DIR}/hsa/include")
+set(SYCL_BUILD_PI_HIP_INCLUDE_DIR "${SYCL_BUILD_PI_HIP_ROCM_DIR}/hip/include" CACHE STRING "HIP include dir")
+set(SYCL_BUILD_PI_HIP_HSA_INCLUDE_DIR "${SYCL_BUILD_PI_HIP_ROCM_DIR}/hsa/include" CACHE STRING "HSA include dir")
```

Comment:
This is not ideal because if a user changes `SYCL_BUILD_PI_HIP_ROCM_DIR`, it won't change these variables since they're cached, so it would still use the previous ROCm installation which would be very confusing.

Could we maybe leave these empty and only actually use them if they're set by users?